src/mm-input-mask/mm-input-mask.scss

```diff
@@ -43,9 +43,10 @@
 	input {
 		font-family: Arimo, sans-serif !important;
 		font-size: 13px !important;
-		line-height: 2.231em !important;
+		line-height: 29px !important;
```

Comment:
Don't recall - but this was never actually working as expected because em needs a 'base' em size (ie `font-size=1em`), which we never actually set. Vetted cross browser and in context, no adverse effects from abandoning em.